apphelpers/socialauth/goog.py

```diff
@@ -11,7 +13,15 @@
     os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "True"
 
 
-def fetch_info(access_token):
-    session = OAuth2Session(token={"access_token": access_token})
-    userinfo = session.get("https://www.googleapis.com/oauth2/v1/userinfo").json()
+def fetch_info(access_token, jwt=False):
```

Comment:
@sayanarijit we can create a separate method
```fetch_info_using_jwt(token):```

Reply:
Need to create prs for all repos again.

Reply:
Updated.